Yoojin-week6

```diff
@@ -0,0 +1,94 @@
+// 문자열 출력하기
+
+#include <stdio.h>
+#include <string.h>
+#define LEN_INPUT 1000001
+
+int main(void) {
+    char s1[LEN_INPUT];
+    scanf("%s", s1);
+
+    int len = strlen(s1);
+    for (int i = 0; i < len; i++){
+        printf("%c", s1[i]);
+    }
+    printf("\n");
+    return 0;
+}
+
```

Comment:
Wow! I think that it is a spoiler!
Later, I will write some comment!

Reply:
Cool